src/shell/components/InAppAnnouncement/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { useState, useMemo } from "react";
+import { ThemeProvider } from "@mui/material";
+import { theme } from "@zesty-io/material";
+import moment from "moment";
+import { useLocalStorage } from "react-use";
+import {
+  Dialog,
+  DialogContent,
+  DialogActions,
+  Typography,
+  Stack,
+  Button,
+  Box,
+} from "@mui/material";
+import OpenInNewRoundedIcon from "@mui/icons-material/OpenInNewRounded";
+import ScheduledRoundedIcon from "@mui/icons-material/ScheduleRounded";
+import PlayArrowRoundedIcon from "@mui/icons-material/PlayArrowRounded";
+
+import { useGetAnnouncementsQuery } from "../../services/marketing";
+
+export const InAppAnnouncement = () => {
+  const [activeAnnouncementIndex, setActiveAnnouncementIndex] = useState(0);
+  const [readAnnouncements, setReadAnnouncements] = useLocalStorage(
+    "zesty:readAnnouncements",
+    []
+  );
+  const { data: announcements } = useGetAnnouncementsQuery();
+
+  const unreadAnnouncements = useMemo(() => {
+    if (announcements?.length) {
+      return announcements
```

Comment:
Currently if there are multiple announcements in the publish range the pop-up will cycle through them as you ignore them. It should only show the single latest announcement.

Reply:
@shrunyan oohh, I thought it was supposed to show everything starting from the latest announcement first.